### DISCUSSION TITLE
Add mark and copy functionality of 2 missing fields in the BSQ section

### DISCUSSION THREAD

**Discussion by teicheld** [bisq-network/bisq#5706] [Ideas]
### Description

Cant mark and copy the right top two of all 6 values in the "BSQ overview section"

#### Version

1.7.3

### Steps to reproduce

only all 3 fields of the left side work and the last of the right side.

### Screenshots

![cant_mark_and_copy](https://user-images.githubusercontent.com/87772815/133920582-83ae5128-e223-4a87-a888-a3f5a04c9094.png)

**Comment by pazza83** [comment]
Hi @teicheld thanks for this.

If someone is able to fix this please can they do the same for all the information on the trade details screen? Often I would like to copy the info and have to manually type it. 

![trade ID](https://user-images.githubusercontent.com/66367547/133944959-2a109dd7-a72b-4c48-aa86-f8d20395c5d6.png)

**Comment by pazza83** [comment]
Would also be good to add this feature to mediation and arbitration tickets.

**Reply by ghost** [reply]
Fixed with https://github.com/bisq-network/bisq/pull/5747
